README.md

```diff
@@ -1,20 +1,42 @@
+Jonathan Erquinigo
 # Class Exercises
 
 1.
   * Create a Human class that takes in a name and age.
-  * Add the function `ageOneYear` that ages the human. 
+  * Add the function `ageOneYear` that ages the human.
   * Add the function `eating`, that logs "mmm, mmm, mmm, I'm love'n it".
   * Create an instance of the Human class.
     * console log your humans age
     * call ageOneYear on your human
     * console log their age again.
     * call eating on your human.
-    
-2. 
+
+    class Human {
+    constructor(name,age){
+      this.name = name;
+      this.age = age;
+      };
+    ageOneYear(){
+      this.age++
+      return this.age++
+      };
+
+    eating(){
+      return "mmm, mmm, mmm, I'm love'n it"
+      };
+
+    };
+
```

Comment:
The eating function asks for you to log the string, not return it. Console.log the string within the function instead.